runtime/config-params.ts

```diff
@@ -1 +1,6 @@
-export type SkyuxConfigParams = string[] | { [key: string]: boolean | { value?: any; required?: boolean } };
```

Comment:
I'm not necessarily opposed, just don't recall talking explicitly about it.  Was the intention here to remove support for an array of strings?

Reply:
@Blackbaud-BobbyEarl I may've misunderstood the issue, but I took it to mean that `SkyConfigParams` would always be considered an object. This would make it easier to document and support, instead of having two different types.

https://github.com/blackbaud/skyux2/projects/9#card-10723415

Reply:
No, you're right.  That's a good catch regarding deprecating that syntax.  Sorry for the confusion.